cibyl/cli/parser.py

```diff
@@ -45,10 +45,21 @@ def parse(self):
         # First item is the namespace of the parsed known arguments
         return self.argument_parser.parse_known_args()[0]
 
-    @staticmethod
-    def extend(ci_environments):
-        """Extends parser with arguments based from the given
-        CI environment entities.
-        """
-        for env in ci_environments:
-            LOG.debug("extending arguments from env %s", env.name.value)
+    def get_group(self, group_name):
+        group = None
+        for action_group in self.argument_parser._action_groups:
```

Comment:
Yeah, you are right. I'll take care of it.

Reply:
Well, after looking into it, it looks like the common behavior is to actually ignore it. Not sure it's worth the effort of writing an additional class if all we do is just access it the same way. Let me know what do you think.

Reply:
IMO creating a new class just for this is not worth it. The only case it might be helpful is if the private variable name is not very clear, to provide a more obvious name

Reply:
Ok, we can continue then with this approach.